File: Numbers/Find all Palindrome Numbers in a given range.py
def palindrome(n):
    revNum = 0
    dup = n
    while(n > 0):
        last_digit = n % 10
        revNum = (revNum * 10) + last_digit
        n = n // 10
    
    if dup == revNum:
        return True
    else:
        return False
    
def main():
    min_val = int(input("Enter the minimum value: "))
    max_val = int(input("Enter the maximum value: "))
    # number = int(input('Enter the number: '))
    # if palindrome(number):
    #     print(number,' is a palindrome')
    # else:
    #     print(number,' is not a palindrome')
        
    for i in range(min_val, max_val + 1):
        if palindrome(i):
            print(i, end=" ")
if __name__ == '__main__':
    main()

File: Numbers/Find all Palindrome Numbers in a given range.cpp
#include <bits/stdc++.h>
using namespace std;

bool palindrome(int n){
    int revNum = 0,dup = n;
    while(n>0){
        int last_digit = n % 10;
        revNum = (revNum * 10) + last_digit;
        n = n / 10;
    }
    if(dup == revNum){
        return true;
    }else{
        return false;
    }
}
int main() {
    int min, max;
    
    // Input range
    cout << "Enter the minimum value: ";
    cin >> min;
    cout << "Enter the maximum value: ";
    cin >> max;
    
    cout << "Palindrome numbers between " << min << " and " << max << " are: ";
    
    // Iterate through the range and print palindrome numbers
    for (int i = min; i <= max; i++) {
        if (palindrome(i)) {
            cout << i << " ";
        }
    }
    cout << endl;
    
    return 0;
}
